docs/src/components/Theming/TokenList.tsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import { useTheme } from '@aws-amplify/ui-react';
+import {
+  SpaceBlock,
+  BorderWidthBlock,
+  FontFamilyBlock,
+  FontSizeBlock,
+  LineHeightBlock,
+  FontWeightBlock,
+  ColorBlock,
+  RadiusBlock,
+} from './TokenBlocks';
+
+type Namespaces =
+  | 'colors'
+  | 'radii'
+  | 'space'
+  | 'borderWidths'
+  | 'lineHeights'
+  | 'fonts'
+  | 'fontSizes'
+  | 'fontWeights';
+
+export function createTokenList(tokens) {
+  // Creates a flattened array out of the token object passed to createTokenList()
+  let tokenList = [];
+  function iterateGroup(group) {
+    Object.values(group).forEach((value) => {
+      if (typeof value === 'object' && value.hasOwnProperty('name')) {
```

Comment:
You can use this util `import { isDesignToken } from '@aws-amplify/ui';` to replace the conditional

Reply:
Oh perfect! Added that to both places.